### DISCUSSION TITLE
Discord bot not authenticating

### DISCUSSION THREAD

**Discussion by klercke** [mozilla/hubs#3303] [Hubs Cloud Questions] [answered]
After a conversation with Elgin today, I was encouraged to post my problem here.
I'm running Hubs Cloud on AWS. I have everything working except the Discord bot. It can create a room, but when I try to authenticate via Discord, it just asks for my Discord credentials again. It also seems unable to properly bridge the channel to a room when I put it in the topic of the channel.

**Comment by robinkwilson** [comment, marked as answer]
Unfortunately the discord bot isn't available on Hubs Cloud at this time. This is certainly an issue we'd like to address in the future, but for now, I'd recommend not using it.

Feel free to follow: https://github.com/mozilla/hubs-cloud/issues/128
